modules/developer-cli-odo-creating-a-database-with-odo.adoc

```diff
@@ -49,7 +49,7 @@ This configuration ensures that when a database service is started, appropriate
 [source,yaml]
 ----
   databaseName: "sampledb"
-  databasePassword: "samplepwd"
+  databasePassword: "password"
   databaseUser: "sampleuser"
```

Comment:
this one changes the pattern of the other values and is not in `<>`. Not sure what to suggest here, but maybe:
```suggestion
  databaseName: "<database_name>"
  databasePassword: "<password>"
  databaseUser: "<username>"
```

Reply:
Updated. Yes, I'm unsure if quotes are needed or not, but I'll retain them to be safe.